zmdocument.cls

```diff
@@ -406,11 +412,27 @@ overlay={\node at (frame.north west) {\createBoxIcon{#5}{#6}};}
 }
 
 %%% TITLE PAGE
-\newcommand{\licence}[1]{\def\@licence{#1}}
+
+\def\@imagesPath{./img}
+\newcommand{\imagesPath}[1]{\def\@imagesPath{#1}}
+\def\@logoPath{\@imagesPath/logo}
+\newcommand{\tutoLink}[1]{\def\@tutoLink{#1}}
+
+\newcommand{\licence}[3][]{%
+   \ifthenelse{\isempty{#1}}%
+   {\def\@licence{\color{white}Licence~:~\@titlePageLink{#2}{#3}}}
+   {\def\@licence{\@titlePageLink{\includegraphics[width=0.5\textwidth]{#1}}{#3}}}
+}
+
 \newcommand{\logo}[1]{\def\@logo{#1}}
 \newcommand{\website}[1]{\def\@website{#1}}
 \newcommand{\editor}[1]{\def\@editor{#1}}
-\newcommand{\editorlogo}[1]{\def\@editorlogo{#1}}
+\newcommand{\editorlogo}[2][]{
```

Comment:
According to [the rule](https://github.com/zestedesavoir/latex-template/blob/master/CONTRIBUTING.md#coding-style), that should be `\editorLogo` :-)